guides/source/association_basics.md

```diff
@@ -1658,7 +1658,7 @@ Controls what happens to the associated objects when their owner is destroyed:
 
 * `:destroy` causes all the associated objects to also be destroyed
 * `:delete_all` causes all the associated objects to be deleted directly from the database (so callbacks will not execute)
-* `:nullify` causes the foreign keys to be set to `NULL`. Callbacks are not executed.
+* `:nullify` causes the foreign key to be set to `NULL`. On polymorphic associations both the foreign key and foreign type columns will be set to `NULL`. Callbacks are not executed.
```

Comment:
Is there any reason to choose different wording in the guide from in the doc?
The list is the same wording without about the on polymorphic associations.

https://github.com/rails/rails/blob/bdf59e5336c8b5dee0f0648a6ef3954da40334b7/activerecord/lib/active_record/associations.rb#L1294-L1299

Reply:
I didn't have any particular reason in my mind @kamipo It's just went this way.

Reply:
If so, I'd prefer to sync the guide and the doc as before.